bee-message/src/output/signature_locked_single.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{address::Address, constants::IOTA_SUPPLY, error::ValidationError};
+
+use bee_packable::{Packable, UnknownTagError};
+
+use core::ops::RangeInclusive;
+
+/// Valid amounts for a signature locked single output.
+pub const SIGNATURE_LOCKED_SINGLE_OUTPUT_AMOUNT: RangeInclusive<u64> = 1..=IOTA_SUPPLY;
+
+/// An output type which can be unlocked via a signature. It deposits onto one single address.
+#[derive(Clone, Debug, Eq, PartialEq, Ord, PartialOrd, Packable)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+#[packable(unpack_error = UnknownTagError<u8>)]
+pub struct SignatureLockedSingleOutput {
+    address: Address,
+    amount: u64,
+}
+
+impl SignatureLockedSingleOutput {
+    /// The output kind of a `SignatureLockedSingleOutput`.
+    pub const KIND: u8 = 0;
```

Comment:
Is there any way we could guarantee that the different `KIND`s never overlap?

Reply:
would a
```rust
#[repr(u8)]
enum OutputKind {
    SignatureLockedSingleOutput = 0,
    ...
}
```
achieve what you want to prevent? At least it would become more obvious this way, right?

Reply:
This is probably the simplest way to achieve this, and it's nice and obvious.

Reply:
What's your opinion on this @thibault-martinez?